src/Psr/CacheItemPool/CacheItem.php

```diff
@@ -3,24 +3,27 @@
 namespace Laminas\Cache\Psr\CacheItemPool;
 
 use DateInterval;
-use DateTimeImmutable;
 use DateTimeInterface;
-use DateTimeZone;
+use Lcobucci\Clock\Clock;
```

Comment:
Could we perhaps use `stella-maris/clock` as interface? That will allow a bit more interoperability as users can then decide which of the implementations they might want to use. As that interface is forward-compatible to the current draft of `psr/clock` that should also make this a future-proof solution. 

As `lcobbuci/clock' implements that interface there is not much change necessary...

Reply:
```suggestion
use StellaMaris\Clock\ClockInterface;
```